src/style/design-tokens/theme-compatibility.util.js

```diff
@@ -46,7 +47,7 @@ export default (theme) => ({
   colorsUtilityReadOnly400: "#F2F5F6",
   colorsUtilityReadOnly500: "#E6EBED",
   colorsUtilityReadOnly600: "#CCD6DB",
-  colorsActionMajor150: "#B3D9C8",
+  colorsActionMajor150: theme.colors.loadingBarBackground, // TODO (design-tokens): update after merge FE-4443
```

Comment:
FE-4443 is merged so this might as well be updated before we merge this one in 👍🏻